### PR TITLE
Add query timeout sqlcmd tests

### PR DESCRIPTION
**Pull request by jxchinaren**

### CODE REVIEW

File: tests/sqlcmd/scripts/querytimeout/timeout.config [hunk 1]
```diff
@@ -0,0 +1 @@
+--query-timeout=1000
```
Comment (wolffcm) [inline tests/sqlcmd/scripts/querytimeout/timeout.config]: Missing newline at end of file.

File: tests/sqlcmd/sqlcmdtest.py [hunk 2]
```diff
@@ -272,15 +272,28 @@ def do_main():
...
                 if not inpath.endswith(".in"):
                     continue
-                print "Running ", os.path.join(parent, inpath)
                 prefix = inpath[:-3]
+
+                config_params = ""
+                prompt = "Running " + os.path.join(parent, inpath)
+                if os.path.isfile(os.path.join(parent, prefix + '.config')):
+                    with open (os.path.join(parent, prefix + '.config'), "r") as configFile:
+                        config_params=configFile.read().replace('\n', '')
```
Comment (wolffcm) [inline tests/sqlcmd/sqlcmdtest.py]: Will this work with more than one option in the .config file?  Seems like you would want to split the contents of the file on the `\n` character (producing a list), and the concatenate that to `['../../sqlcmd']` in the call to `subprocess.call` below.

**Comment by wolffcm** [comment]
That's all my comments.

File: tests/sqlcmd/sqlcmdtest.py (hunk 2 above)
Comment (jxchinaren) [inline tests/sqlcmd/sqlcmdtest.py]: Yes, it's only working for 1 configuration. I added the multiple configuration support.

**Comment by wolffcm** [comment]
Looks good to me.

**Pull request merged**